core/modules/system/system.install

```diff
@@ -3421,6 +3421,21 @@ function system_update_1088() {
   }
 }
 
+/**
+ * Enable Admin Bar's Language Switcher component if this is a multilingual
+ * site.
+ */
+function system_update_1089() {
+  if (!module_exists('locale') && language_multilingual()) {
```

Comment:
This conditional seems odd to me 🤔 ...shouldn't we be checking whether the Locale module is enabled and `language_multilingual()` is `TRUE` (and then enabling the language switcher in that case)?

If we want to be returning early, then we should be doing it when the Locale module is disabled OR if the site isn't multilingual. Right?